### ISSUE TITLE
stylelint dont working properly with scss

### ISSUE THREAD

**Issue by AesthetiCoder** [stylelint/stylelint#6819]
### What steps are needed to reproduce the bug?

i have a normal config of stylint and i get an error of customSyntax when linting i have a normal scss file and get that error

`you should use the "customSyntax" option when linting something other than CSS`

### What Stylelint configuration is needed to reproduce the bug?

you can reproduce the error running this stylelint config 

```json
{
  "extends": "stylelint-config-standard-scss"
}
```

in the output of the vscode console i get the error, this dont happen with all the files, stylelint can work good with other files of mi projects, the file that have problems is normalize.scss from https://necolas.github.io/normalize.css/

i dont want to put a lot of code here to make this issue simple but other file stylelint has problem is my _reset.scss here is the file 

```scss
html,
body,
p,
ol,
ul,
li,
dl,
dt,
dd,
blockquote,
figure,
fieldset,
legend,
textarea,
pre,
iframe,
hr,
h1,
h2,
h3,
h4,
h5,
h6 {
  margin: 0;
  padding: 0;
}

h1,
h2,
h3,
h4,
h5,
h6 {
  font-size: 100%;
  font-weight: normal;
}

ul {
  list-style: none;
}

button,
input,
select {
  margin: 0;
}

html {
  box-sizing: border-box;
}

*,
*::before,
*::after {
  box-sizing: inherit;
}

img,
video {
  max-width: 100%;
  height: auto;
}

iframe {
  border: 0;
}

table {
  border-spacing: 0;
  border-collapse: collapse;
}

td,
th {
  padding: 0;
}
```

also this happends if i do a normal config of stylelint like this, with plugins or not the output console throw the same error

```json
  "plugins": ["stylelint-scss", "stylelint-order"],
  "customSyntax": "postcss-scss",
  "rules": {}
```

### How did you run Stylelint?

i just let my vscode format the files when saving the files, when i run 

```shell
npx stylelint "**/*.scss"
```

i dont get any output in the consele or any error

### Which version of Stylelint or dependencies are you using?

postcss: 8.4.21
postcss-scss:  4.0.6
stylelint: 15.6.0

### What did you expect to happen?

not to have this output in the consele maybe can be an error of the vscode extension that dont detect the customSyntax or is just somehing of the file normalize.scss and reset.scss

### What actually happened?

`you should use the "customSyntax" option when linting something other than CSS`

**Comment by ybiquitous** [comment]
@LuisCoderDev Thanks for the report with reproduction details. But unfortunately, I cannot reproduce the problem with [this demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIAWAXAtgGwDQB0B2ARtAJ654AOZE2+ArjXhgJZlQNQqtRkEYRgBrAI60IKGGQBmTAOa0AThPzSYGKAGcYnfBhgyYebvnEAPFAENF5suUVkmk+ebRK8SeWSQBGTwCZPAMyeACyeAKyeAGwABMD40dFoljJMePDRAAwA3PHR5OZQUKky6dn4AL74+N5+gSHhUbG5khB4KAC06kwAXjDpXhkZAKQ5eAktbe0A7jCyqOl4EPJJGKOVeHQYTWPRzOod+8S6C60wa1WEtCgorfYUV2SaumAo2wlJ8ilpmecbbuhbOI7IgmTo9YrpIjyWDydog374ABUZER8HgBBgLTsSLR5kk4nkb2iILB3Qh0VSSBg8iYKAReCYaBkZAAbkxYBAiUlQVN2SgkP1BiNclS5ih0uYrhB6Q4nC4iVCYaV6RZeDAFUsYZ18mByWVgZrqe1IBgMOZyJp0iazRazhULigjG0kET8oU9WsQFgQNJdAA5ZxwRAwEzOci6AB06jA6nUXvArWkMgQICBCRwIBD4kM6gz6Qzh10zEmkDwSc6FkMlignRjuZAFXjpaTADElkkUCmAFbqVrx2AWlNp6IFlBHVSpFB5kcgM3ifYZsjpkCFiclxOyCvmKvQ2ux6cZucwBcNvDlEDlIA) using `stylelint-config-standard-scss`.

Do I miss something in the demo? Or do you have any suggestions about reproduction?

**Comment by jeddy3** [comment]
Closing as stale. You're welcome to reopen with a reproducible example.

**Comment by AesthetiCoder** [comment]
i found that the mistake is made by a config i have in vscode

```json
  "[scss]": {
    "editor.defaultFormatter": "stylelint.vscode-stylelint"
  },
```

when i have this i get the error in the otput terminal of vscode.

`you should use the "customSyntax" option when linting something other than CSS`
